linux-mister/PKGBUILD

```diff
@@ -78,6 +78,7 @@ _package() {
   cd "${srcdir}/${_srcname}"
 
   KARCH=arm
+  ARCH=arm
```

Comment:
Are you sure this does anything? You only declared a local variable in this function, and you're not using it anywhere. None of the subprocesses (like make) can see it.

Reply:
It is not, sorry. The only change should in pkgrel.